pipe_anchorages/transforms/voyages_create.py

```diff
@@ -0,0 +1,62 @@
+import apache_beam as beam
+import datetime as dt
+import pytz
+
+def create_voyage(visits):
+    # first voyage start from 'we do not know' = None
+    visit_before = {2: None, 3: None, 4: None}
```

Comment:
very small comment (feel free to ignore this) but I think `previous_visit` is a better name than `visit_before`.